Pod/Classes/ios/NYTPhotoViewController.m

```diff
@@ -157,8 +157,8 @@ - (void)didDoubleTapWithGestureRecognizer:(UITapGestureRecognizer *)recognizer {
     CGPoint pointInView = [recognizer locationInView:self.scalingImageView.imageView];
     
     CGFloat newZoomScale = self.scalingImageView.maximumZoomScale;
-    
-    if (self.scalingImageView.zoomScale >= self.scalingImageView.maximumZoomScale) {
+
+    if ((self.scalingImageView.zoomScale - self.scalingImageView.maximumZoomScale) >= 0.0) {
```

Comment:
The change looks fine. However commenting out the if clause entirely causes no unit test failures. Expand the test coverage to include this?